src/components/Inputs/Input/Input.module.scss

```
@@ -0,0 +1,49 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.label {
+  font-size: 1.125rem;
+  font-weight: 500;
+
+  .required {
+    color: $color-violet-5534da;
+  }
+}
+
+.inputContainer {
+  display: inline-flex;
+  border: 0.0625rem solid $color-gray-d9d9d9;
+  border-radius: 0.5rem;
+  width: min(100%, 32.5rem);
+  height: 3rem;
+  padding: 0 1rem;
+  gap: 0.5rem;
+
+  &.error {
+    border-color: $color-red-d6173a;
+  }
+
+  &:focus-within {
+    border-color: $color-violet-5534da;
+  }
+}
+
+.input {
+  outline: none;
+  border: none;
```

```suggestion
  outline: none;
  border: none;
```

요거 아까 위에 datePicker 에도 있던 것 같은데 reset.scss에 넣어도 좋을 것 같네요🙃

---

반영했습니다!